src/browser/api/external_window.ts

```diff
@@ -39,18 +44,44 @@ export function closeExternalWindow(identity: Identity): void {
   externalWindow.forceExternalWindowClose();
 }
 
-export async function disableExternalWindowUserMovement(identity: Identity): Promise<void> {
-  const externalWindow = getExternalWindow(identity);
+export async function disableExternalWindowUserMovement(target: Identity, requestor?: Identity): Promise<void> {
+  const externalWindow = getExternalWindow(target);
   const injectionBus = getInjectionBus(externalWindow);
   await injectionBus.set({ userMovement: false });
   externalWindow.emit('user-movement-disabled');
+
+  // Register a subscription that will fire every time requesting identity closes
+  // and if all requestors for disabling frame are gone - enable the frame.
+  if (requestor) {
+    const key = getKey(externalWindow);
+    const subscriptionKey = `disable-external-window-user-movement-${key}`;
+    const onRequestorClose = async () => {
+      let requestorCount = disabledUserMovementRequestorCount.get(key) || 0;
+      if (requestorCount > 1) {
+        disabledUserMovementRequestorCount.set(key, --requestorCount);
+      } else {
+        await enableExternaWindowUserMovement(target);
```

Comment:
one other way of removing the decrement from enableExternaWindowUserMovement would be to call it here when `requestorCount === 1`